app/templates/partials/summary/date.html

```diff
@@ -1 +1,5 @@
-{{answer.value|pretty_date_range}}
+{% if question.type == 'DateRange' %}
```

Comment:
It would be nice if we could pull the formatting out into different templates so we didn't have this logic in the template